src/components/Card.js

```diff
@@ -73,56 +80,71 @@ export const Card = () => {
         >
           <QRCode
             className="qrcode"
-            style={{ paddingRight: portrait ? '' : '1em' }}
+            style={{
+              paddingRight: portrait ? '' : '1em',
+              height: currentSize + 'px',
+              width: currentSize + 'px',
+            }}
             value={qrvalue}
-            size={175}
+            size={currentSize}
           />
 
-          <div className="inputs">
-            <label>{t('wifi.name')}</label>
-            <textarea
-              id="ssid"
-              type="text"
-              maxLength="32"
-              placeholder={t('wifi.name.placeholder')}
-              autoComplete="off"
-              autoCorrect="off"
-              autoCapitalize="none"
-              spellCheck="false"
-              value={network.ssid}
-              onChange={(e) => setNetwork({ ...network, ssid: e.target.value })}
-            />
-            <label
+          <div>
+            <div
```

Comment:
From a semantics standpoint, it doesn't make much sense to have two nested divs that lack any id or class:

```
<div>
  <div>
```

Whereas before there was a single div labeled "inputs" which told the reader _this is a section of inputs_.

Reply:
Yes you are right. During development I had a situation where the view broke and this div fixed the situation. But now everything is actually working fine. I forgot to delete this div to check how the site behaves after development. Done. Thanks!